src/app/components/universal/newSafetyDocumentUpdate/newSafetyDocumentUpdate.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+
+@Component({
+  selector: 'app-newSafetyDocumentUpdate',
+  templateUrl: './newSafetyDocumentUpdate.component.html',
+  styleUrls: ['./newSafetyDocumentUpdate.component.css']
```

Comment:
Please change these 3 to be snake-case, like `app-new-safety-documents-update`.

I'm also wary of using "new" unless it makes sense. Is it called new because it displays newly updated info? If so then I think "new" should be fine in this context.

Reply:
Don't forget you'll need to update the tag in the homepage html file.